python/dgl/distributed/rpc.py

```diff
@@ -338,6 +550,73 @@ def send_request(target, request):
     msg = RPCMessage(service_id, msg_seq, client_id, server_id, data, tensors)
     send_rpc_message(msg)
 
+def recv_request(timeout=0):
```

Comment:
how is this different from `recv_response`?

Reply:
They will use different class for serilaize/de-serialize